OCR/frontend/e2e/App.spec.ts

```diff
@@ -15,3 +16,67 @@ test('has new template button', async ({ page , baseURL}) => {
     expect(page.url()).toEqual(`${baseURL}/new-template/upload`)
 
 });
+
+test.describe('when templates exist', async () => {
+    test.beforeEach(async ({page}) => {
+        await page.goto('/')
+        await page.evaluate(() => {
+            const templates = [
+
+                {
+                    name: "MumpsQuestV1",
+                    lab: "Quest",
+                    createdBy: "J.Smith",
+                    status: "Completed",
+                    lastUpdated: new Date(Date.parse("2025-03-24"))
+                },
+                {
+                    name: "LBTIRadar",
+                    lab: "Radar",
+                    createdBy: "C.Alex",
+                    status: "Completed",
+                    lastUpdated: new Date(Date.parse("2025-05-30"))
+                },
+                {
+                    name: "COVIDBaylor1",
+                    lab: "Emory",
+                    createdBy: "A.Bryant",
+                    status: "Completed",
+                    lastUpdated: new Date(Date.parse("2025-06-21"))
+                },
+                {
+                    name: "COVIDEMory",
+                    lab: "Baylor",
+                    createdBy: "D.Smith",
+                    status: "Completed",
+                    lastUpdated: new Date(Date.parse("2024-06-21"))
+                },
+            ];
+            localStorage.setItem('templates', JSON.stringify(templates))
+        })
+    })
+    test('displays list of templates if they exist and sorting functions', async ({page, baseURL}) => {
+        await page.goto('/')
+        await expect(page.getByRole('heading', { name: 'Saved Templates' })).toBeVisible()
+        await expect(page.locator('tbody').getByRole('row')).toHaveCount(4)
+        await page.getByText('Updated On').click()
+        await expect(page.locator('tbody').locator('tr').nth(0).getByRole('cell').nth(1)).toHaveText('6/21/2024')
```

Comment:
this fails for me with the following error 

Locator: locator('tbody').locator('tr').first().getByRole('cell').nth(1)
Expected string: "6/21/2024"
Received string: "6/20/2024"

Reply:
Ok, so after some digging - because CI (and my local tests) run in docker containers which don't have their locale set this is a timezone error. Adding a fix for this in a follow up PR.